scripts/firedrake-install-conda.sh

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bash
+
+# For some odd reason, on MacOS we need to tell it where to find `ar`,
+# otherwise it might try to find an `llvm-ar` that doesn't necessarily exist.
+if [[ $(uname -s) == Darwin ]]; then
+    export AR=/usr/bin/ar
+fi
+
+# Make an environment variable for the directory where the conda environment
+# will live; this will depend on whether or not you're using `pyenv`.
+if [[ $(command -v pyenv) ]]; then
```

Comment:
Does this test if we're using pyenv or just if it's installed?

Reply:
It's testing if calling `pyenv` returns a successful exit code, which to me implies that it's actually being used. It would be different if we checked for the existence of the directory `~/.pyenv`, which could just be leftover from a previous installation.

In any case, I think I see where you're going. I could add a `--prefix` argument to the script so you can decide where the conda environment will get built. If you don't give an argument for this, it'll default to one directory if you're using pyenv and another if you're not. Is that what you were thinking?

Reply:
I think that that's a better solution. pyenv returning successfully just means, I think, that pyenv is installed. It doesn't imply that the user intended to use it on this occasion.